dapps/digital-twin-detail/src/components/data/DetailData.ts

```diff
@@ -0,0 +1,44 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+// vue imports
+import Component, { mixins } from 'vue-class-component';
+
+// evan.network imports
+import { EvanComponent } from '@evan.network/ui-vue-core';
+
+@Component
+export default class DetailDataComponent extends mixins(EvanComponent) {
+  navItems = [
+    {
+      key: 'general'
+    },
+    {
+      key: 'specifications'
+    },
+    {
+      key: 'logs'
+    }
+  ].map(entry => {
+    return {
+      label: `_twin-detail.data.${entry.key}.${entry.key}-title`,
```

Comment:
that's maybe bit too generic here? why not add `to` and  `label` property directly to navItems Object? Would also provide the possibility to use translation key outside of `_twin-detail.data.`

Reply:
Well, it's following our current i18n structure. Sure we could "hardcode" it to be more declarative. I don't understand the argument of translations outside of _twin-detail.data tho? Those aren't supposed to happen here to begin with.

Reply:
At this point, you can use every translation key, you want. It's not prescribed by anyone ;)